template/build.gradle.kts

```diff
@@ -26,13 +26,18 @@ allprojects {
     }
 }
 
+configurations.all {
+    resolutionStrategy.cacheChangingModulesFor(0, TimeUnit.SECONDS)
+    resolutionStrategy.cacheDynamicVersionsFor (0, TimeUnit.SECONDS)
+}
+
```

Comment:
Would you mind giving some explanation on why this is required? 🙇

Reply:
@Tuubz It is used for Gradle cache change, my prasice when trying to resolve Jacoco coverage report. Now we dont need it anymore 🙏